stage/services/check_links/main.tf

```diff
@@ -0,0 +1,41 @@
+resource "aws_cloudwatch_event_rule" "check-links" {
+  name                = "check-links"
+  description         = "Run check-links API call via cron"
+  schedule_expression = "cron(0 1 * * ? *)"
```

Comment:
I like to avoid running these jobs on the hour, but rather at an arbitrary minute, so that multiple jobs don't compete for resources (mostly Solr queries right now).